src/Compilers/Core/Portable/Emit/CommonPEModuleBuilder.cs

```diff
@@ -763,7 +763,7 @@ public ImmutableArray<ISymbolInternal> GetAllMembers()
 
                 if (NestedTypes != null)
                 {
-                    foreach (var type in NestedTypes)
+                    foreach (var type in NestedTypes.OrderBy(t => t.Name, StringComparer.Ordinal))
```

Comment:
>NestedTypes.OrderBy(t => t.Name, StringComparer.Ordinal) [](http://example.com/codeflow?start=41&length=56)

Consider adding a `NestedTypesOrdered()` method to this type, for use here and below.